src/F2_autoreport.py

```diff
@@ -189,6 +193,31 @@ def make_dash_data_table(df, title=None):
 
 
 def insert_subsection(title, content, **kwargs):
+    r"""
+    Inserts sub-sections within the Dash app layout, such as Input data, simulation results, etc.
+
+    Parameters
+    ----------
+    title : str
+        This is the title or heading of the subsection.
+
+    ...
+
+    content : list
+        This is typically a list of html elements or function calls returning html elements, that make up the
+        body of the sub-section.
+
+     ...
+
+    kwargs : Any possible optional arguments such as styles, etc.
+
+    Returns
+    -------
+    html.Div()
```

Comment:
Not sure if you can define a variable name for this and say that it is type html.Div().

Reply:
Mabye the type is also something like a plotly dash html formatting section?

Reply:
I feel it would be better to leave it like this as the plots are indeed returned in a html.Div() element instead of a variable?

Reply:
Okay!